src/EPSG_2056.js

```diff
@@ -4,7 +4,7 @@
 import somerc from './somerc.js';
 import {create} from './utils.js';
 
-const code = 'EPSG:2056';
+export const code = 'EPSG:2056';
```

Comment:
if the code if exported here, shouldn't `export default code;` be removed?

Reply:
Ideally yes, but my understanding is that individual files are part of the API (because library users can import them directly), so that would be a breaking change.